ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -46,6 +46,7 @@ statement
     | PRINT (identifier| STRING) printClause                                #printTopic
     | (LIST | SHOW) QUERIES EXTENDED?                                       #listQueries
     | TERMINATE QUERY? identifier                                           #terminateQuery
```

Comment:
Should we just eliminate the `TERMINATE QUERY <query>` grammar in favor of using `TERMINATE <query>`? I don't think we need two ways to do the same thing here, and `TERMINATE <query>` feels more consistent with `TERMINATE ALL`.

Reply:
the `QUERY` is optional today (you don't need it)

Reply:
Happy to.